shared/src/business/utilities/abbreviateState.test.js

```diff
@@ -0,0 +1,12 @@
+const { abbreviateState } = require('./abbreviateState');
+
+describe('abbreviateState', () => {
```

Comment:
Might it be worth adding a test around multi-word cities? e.g. 'Las Vegas, Nevada'

Reply:
Ooh thats a great idea, on it!